gpu/install_gpu_driver.sh

```diff
@@ -223,6 +223,34 @@ EOF
   systemctl --now enable gpu-utilization-agent.service
 }
 
+function config_gpu_exclusive_mode {
+  # check if running spark3, if not, enable GPU exclusive mode
+  spark-submit --version &> /tmp/version.txt
+  if ! grep -q "version 3" "/tmp/version.txt" ; then
```

Comment:
You can use `sed` command to parse version in a variable instead of a temp file:
```bash
local spark_version
spark_version=$(spark-submit --version 2>&1 | sed -n 's/.*version[[:blank:]]\+\([0-9]\+\.[0-9]\).*/\1/p' | head -n1)
if [[ ${spark_version} == 3.* ]]; then
  ...
fi
```

Reply:
Made the change following suggestion